plugins/task-resources/src/components/state/StatesEditor.svelte

```diff
@@ -15,25 +15,34 @@
 -->
 <script lang="ts">
   import { Class, Ref } from '@hcengineering/core'
-  import { AttributeEditor, getClient } from '@hcengineering/presentation'
-  import type { DoneState, State } from '@hcengineering/task'
+  import { AttributeEditor, AttributesBar, getClient } from '@hcengineering/presentation'
+  import type { DoneState, KanbanTemplate, State } from '@hcengineering/task'
   import {
     CircleButton,
     IconAdd,
     IconMoreH,
     Label,
     showPopup,
     getPlatformColor,
-    eventToHTMLElement
+    eventToHTMLElement,
+    EditBox,
+    Icon,
+    Button,
+    Component
   } from '@hcengineering/ui'
   import { createEventDispatcher } from 'svelte'
   import { ColorsPopup } from '@hcengineering/view-resources'
+  import { StyledTextBox } from '@hcengineering/text-editor'
+  import tracker from '@hcengineering/tracker'
+  import { getFiltredKeys } from '@hcengineering/view-resources/src/utils'
   import Circles from './Circles.svelte'
   import StatusesPopup from './StatusesPopup.svelte'
   import task from '../../plugin'
   import Won from '../icons/Won.svelte'
   import Lost from '../icons/Lost.svelte'
+  import recruit from '../../../../recruit-resources/src/plugin'
```

Comment:
Please remove it and move `RelatedIssues` string to task plugin

Reply:
I also use `recruit.class.Vacancy` from recruit package in this module. I think this package should be added to task-resources. Isnt it?

Reply:
I think we should create some optional components contributing in models as field or mixing to KanbanTemplateSpace. Also I think we need create new related issue for each vacancy.